subprojects/test-runner/client/src/main/kotlin/com/avito/runner/scheduler/runner/TestRunnerImpl.kt

```diff
@@ -97,13 +106,12 @@ internal class TestRunnerImpl(
                     )
 
                     testSuiteListener.onTestSuiteFinished()
-                    logger.info("Test run end successfully")
+                    logger.info("Test run finished successfully")
                     Result.Success(result)
-                } catch (e: TimeoutCancellationException) {
-                    logger.critical("Test run end with timeout", e)
-                    Result.Failure(e)
                 } catch (e: Throwable) {
-                    logger.critical("Test run end with error", e)
+                    logger.critical("Test run finished with ${
+                        if (e is TimeoutCancellationException) "timeout" else "error"
```

Comment:
The commodity is not an option because of poor readability
What Is a problem to make a separate log when Failure for Timeout is created?